bitmap-team-deuces/gulpfile.js

```diff
@@ -0,0 +1,18 @@
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('runTests', () => {
+  gulp.src(__dirname + '/test/*.js')
+  .pipe(mocha());
+});
+
+gulp.task('runLint', () => {
+  return gulp.src(['**/*.js','!node_modules/**'])
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failAfterError());
+});
+
+gulp.task('default', ['runTests', 'runLint'], () => {
```

Comment:
Remove the anonymous arrow function at the end of this line, it doesn't do anything and does not need to be there.